packages/dev-approuter/package.json

```diff
@@ -16,13 +16,5 @@
     "cds-plugin-ui5": "workspace:^",
     "express": "^4.21.2",
     "ui5-middleware-cap": "workspace:^"
-  },
-  "peerDependencies": {
```

Comment:
There are no dependencies to @sap/cds anymore?

Reply:
Correct, only as a second level dependency via the `ui5-middleware-cap`.